test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -577,4 +594,31 @@ protected static boolean aliasExists(String index, String alias) throws IOExcept
         assertNotNull(responseEntity);
         return responseEntity;
     }
+
+    /**
+     * Is this template one that is automatically created by xpack? Deleting
+     * these templates doesn't hurt anything but it makes the tests run more
+     * slowly because it then has to wait for the templates to be recreated.
```

Comment:
this comment seems kind of odd for a true/false isXXX method :) It cares not about a delete.